lib/lightning/policies/workflows.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Lightning.Policies.Workflows do
+  @moduledoc """
+  The Bodyguard Policy module for Workflows API.
+  """
+  @behaviour Bodyguard.Policy
+
+  alias Lightning.Accounts.User
+  alias Lightning.Projects.Project
+  alias Lightning.Run
+
+  @type actions :: :access_write | :access_read
```

Comment:
The other bodyguard polices name the specific actions to be done, could we do the same here as well? 
Something like `:create_workflow`, `:update_workflow` ...

Reply:
From collections we've started using this pattern of access to make it more noticeable and different from the operation themselves (borrowed from ACL idea).  (https://github.com/OpenFn/lightning/blob/8e082b20e60e072d0bcd2cdb189e541a41d6461c/lib/lightning/policies/collections.ex#L16)

Another reason is that the `:access_write` is the same permission check for `:create_workflow` and `:updated_workflow`.